bottlecap/src/config/mod.rs

```diff
@@ -255,6 +272,22 @@ pub mod tests {
         });
     }
 
+    #[test]
+    fn test_fallback_on_otel() {
+        figment::Jail::expect_with(|jail| {
+            jail.clear_env();
+            jail.set_env("DD_EXTENSION_VERSION", "next");
+            jail.set_env(
+                "DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_HTTP_ENDPOINT",
```

Comment:
There's also a complementary `DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT` we should probably listen for as well.

Reply:
Its in the config file as fallback too